src/main/java/com/mraof/minestuck/entity/item/BarbasolBombEntity.java

```diff
@@ -0,0 +1,56 @@
+package com.mraof.minestuck.entity.item;
+
+import com.google.common.graph.Network;
+import com.mraof.minestuck.item.MSItems;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.IRendersAsItem;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.projectile.ProjectileItemEntity;
+import net.minecraft.item.Item;
+import net.minecraft.network.IPacket;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.world.Explosion;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.network.NetworkHooks;
+
+public class BarbasolBombEntity extends ProjectileItemEntity implements IRendersAsItem
+{
+    private LivingEntity throwerIn;
```

Comment:
Looks unused, except for line 40, which looks like an accidental use considering that the thrower here is never set